qa/tasks/quiescer.py

```diff
@@ -186,11 +187,15 @@ def dump_ops_all_ranks(self, dump_tag):
 
             self.logger.debug(f"Dumping ops on rank {rank} ({name}) to a remote file {remote_path}")
             try:
-                _ = self.fs.rank_tell(['ops', '--flags=locks', f'--path={daemon_path}'], rank=rank)
+                p = self.fs.rank_tell(['ops', '--flags=locks', f'--path={daemon_path}'], rank=rank, wait=False)
+                procs.append(p)
                 remote_dumps.append((info, remote_path))
             except Exception as e:
                 self.logger.error(f"Couldn't execute ops dump on rank {rank}, error: {e}")
 
+        for p in procs:
+            p.wait()
+
         # now get the ops from the files
         for info, remote_path in remote_dumps:
```

Comment:
and then insert the `p.wait` call in this loop under the `try:` to collect at least some of the dumps if one fails

Reply:
done